tests/unit/test_mindful.py

```diff
@@ -0,0 +1,90 @@
+import os
+from unittest import mock
+from datetime import datetime, timedelta
+
+import pytest
+
+# from viadot.config import local_config
```

Comment:
Remove

Reply:
I added a decorator from this library: `@pytest.fixture` it can't be removed.